_docs-2/troubleshooting/advanced.md

```diff
@@ -333,16 +333,21 @@ overridden by using the `--local-wal-directories` option on the tool. It can be
 **I am trying to start the manager after upgrading but the upgrade is aborting with the following message:**
   `org.apache.accumulo.core.client.AccumuloException: Aborting upgrade because there are outstanding FATE transactions from a previous Accumulo version.`
 
-You can use the shell to delete completed FATE transactions using the following:
+For versions 2.1 and later, you can use the admin fate command to delete completed FATE transactions using the following:
 
 * Start tservers
 * Start shell
-* Run `fate print` to list all
-* If completed, just delete with `fate delete`
+* Run `accumulo admin fate --print` to list all transactions
+* If the transactions have completed, just delete with them with `accumulo admin fate --delete TXID [TXID...]`
 * Start managers once there are no more fate operations
 
 If any of the operations are not complete, you should rollback the upgrade and troubleshoot completing them with your prior version.
 
+Prior to 2.1, the same step apply, but the fate command is accessed with the Accumulo shell instead of the admin command.  The shell commands are as follows: 
```

Comment:
```suggestion
Prior to 2.1, the same steps apply, but the fate command is accessed with the Accumulo shell instead of the admin command.  The shell commands are as follows: 
```